scripts/publish-circleci.js

```diff
@@ -15,83 +26,25 @@ shell.set('+v');
  * Run this script with SALESFORCEDX_VSCODE_VERSION, CIRCLECI_TOKEN & CIRCLECI_BUILD as environment variables
  * i.e. SALESFORCEDX_VSCODE_VERSION=x.y.z ./scripts/publish-circleci.js
  *
+ * To run a prevalidation for the environment variables run `node scripts/publish-circleci.js -- -v`.
  */
 
-// Checks that you are running this with Node v12.4.0 and above
-const [version, major, minor, patch] = process.version.match(
-  /^v(\d+)\.?(\d+)\.?(\*|\d+)$/
-);
-if (parseInt(major) !== 12 || parseInt(minor) < 4) {
-  console.log(
-    'You do not have the right version of node. We require version 12.4.0.'
-  );
-  process.exit(-1);
-}
-// Check if you have installed all the required tooling
-if (!shell.which('vsce')) {
-  console.log('vsce is not installed or could not be found');
-  process.exit(-1);
-}
-
-if (!shell.which('lerna')) {
-  console.log('lerna is not installed or could not be found');
-  process.exit(-1);
-}
-
-if (!shell.which('aws')) {
-  console.log('aws cli is not installed or could not be found');
+if (process.argv.indexOf('-v') > -1) {
+  console.log('Running prevalidation of environment variables.');
+  console.log(`SALESFORCEDX_VSCODE_VERSION: ${process.env['SALESFORCEDX_VSCODE_VERSION']}`);
+  console.log(`CIRCLECI_TOKEN: ${process.env['CIRCLECI_TOKEN']}`);
+  console.log(`CIRCLECI_BUILD: ${process.env['CIRCLECI_BUILD']}`);
   process.exit(-1);
```

Comment:
Should the exit status here be -1 ?

Reply:
No, probably not. I will look for one that is more appropriate.